pkg/interface/src/apps/chat/components/lib/message.js

```diff
@@ -4,6 +4,43 @@ import { uxToHex, cite, writeText } from '../../../../lib/util';
 import moment from 'moment';
 import ReactMarkdown from 'react-markdown';
 import RemarkDisableTokenizers from 'remark-disable-tokenizers';
+import SyntaxHighlighter from 'react-syntax-highlighter';
+import { darcula, atomOneLight } from 'react-syntax-highlighter/dist/esm/styles/hljs';
+
```

Comment:
This ought to be in a seperate file.

Reply:
and if you're feeling generous, typescript would be nice but not a blocker for this to be merged.

Reply:
Got it. Move `CodeBlock` into a separate file?

Reply:
Yep!